js/demo-app/public/index.html

```diff
@@ -24,8 +24,8 @@
         <div class="nav">
             <span>Programmatically:</span>
             <br/>
-            <button class="nav-item" onclick="setRoute('#/one')">Component 1</button>
-            <button class="nav-item" onclick="setRoute('#/two')">Component 2</button>
+            <button class="nav-item" onclick="setRoute('/one')">Component 1</button>
+            <button class="nav-item" onclick="setRoute('/two')">Component 2</button>
```

Comment:
This is a secondary effect of the change to navigation request payloads, which resulted in a new `RouteChange` message type in `AppToHost.elm`